exercises/perfect-numbers/perfect_numbers_test.go

```diff
@@ -0,0 +1,48 @@
+package perfect
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestPositiveNumberRequiredErrorDefined(t *testing.T) {
+	if !reflect.TypeOf(ErrOnlyPositive).Implements(reflect.TypeOf((*error)(nil)).Elem()) {
```

Comment:
No. IMHO, we don't need neither of them.

However, we want to ensure so I tried to make it as a separate achievement and put as the first test but I can change it to compile time check, as you wish.